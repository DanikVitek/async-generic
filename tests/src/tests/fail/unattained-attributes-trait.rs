use async_generic::async_generic;
use async_trait::async_trait;

#[async_generic(
    #[async_trait]
)]
trait A {}

fn main() {}