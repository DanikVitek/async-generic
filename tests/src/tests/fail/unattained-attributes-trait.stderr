error: unexpected end of input, expected `sync_trait` or `async_trait`
 --> src/tests/fail/unattained-attributes-trait.rs:4:1
  |
4 | / #[async_generic(
5 | |     #[async_trait]
6 | | )]
  | |__^
  |
  = note: this error originates in the attribute macro `async_generic` (in Nightly builds, run with -Z macro-backtrace for more info)

error: attributes must be placed on `sync_trait` and/or `async_trait`
 --> src/tests/fail/unattained-attributes-trait.rs:5:5
  |
5 |     #[async_trait]
  |     ^

warning: unused import: `async_trait::async_trait`
 --> src/tests/fail/unattained-attributes-trait.rs:2:5
  |
2 | use async_trait::async_trait;
  |     ^^^^^^^^^^^^^^^^^^^^^^^^
  |
  = note: `#[warn(unused_imports)]` on by default
